Tests/AfluentTests/ErrorsTests/TimeoutErrorTests.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  TimeoutErrorTests.swift
+//  Afluent
+//
+//  Created by Dalton Alexandre on 4/5/25.
+//
+
+
+import Testing
+@testable import Afluent
+
+struct TimeoutErrorTests {
+    @Test func testTimeoutErrorEquality() async {
+        let error1 = TimeoutError.timedOut(duration: Duration.seconds(5))
+        let error2 = TimeoutError.timedOut(duration: Duration.seconds(10))
+        #expect(error1 == error2, "TimeoutError equality should not depend on duration")
+    }
+
+    @Test func testTimeoutErrorDescription() async {
+        let duration = Duration.seconds(3)
+        let error = TimeoutError.timedOut(duration: duration)
+        #expect((error.errorDescription != nil), "Timed out after waiting \(duration)")
+    }
```

Comment:
This expectation likely doesn't quite do what you wanted. It asserts the description isn't nil, but it does not assert the contents of the description. Instead I recommend something like this:

```suggestion
    @Test func testTimeoutErrorDescription() async throws {
        let duration = Duration.seconds(3)
        let error = TimeoutError.timedOut(duration: duration)
        let description = try #require(error.errorDescription)
        #expect(description == "Timed out after waiting \(duration)")
    }

```

Reply:
Thank you